docs/buildah-config.md

```diff
@@ -65,6 +65,13 @@ its OS is kept, otherwise the host's OS's name is recorded.
 Specifies a *port* to expose when running containers based on any images which
 will be built using the specified container.
 
+**--shell** *shell*
+
+Specify the *shell* to run inside of the container image
+The shell instruction allows the default shell used for the shell form of commands to be overridden. The default shell command to run Linux is "/bin/sh -c".
```

Comment:
That second sentence would probably read better as 'The default shell for Linux containers is "/bin/sh -c"'.

Reply:
Yes it does.  The urvave only gets confused when it is "-c", it does not complain with "bash -c"